src/app/scripts/cac/control/cac-control-directions.js

```diff
@@ -239,6 +226,18 @@ CAC.Control.Directions = (function (_, $, moment, Control, Geocoder, Routing, Ty
         directionsListControl.hide();
     }
 
+    /**
+     * Wipe out user-set trip options and clear controls
+     */
+    function clearUserSettings() {
```

Comment:
I think this should be a function in the Home page controller that triggers on a `tabShown` event, but either setup will cause conflicts with #699, so we can deal with it then.